.storybook/webpack.config.js

```diff
@@ -2,6 +2,11 @@ const path = require("path");
 const pathToInlineSvg = path.resolve(__dirname, "../src/common/svgs/");
 
 module.exports = ({ config }) => {
+  config.resolve.alias = {
```

Comment:
this is needed, otherwise storybook doesn't know how to handle `import foo from 'src/...`